src/renderer/atlas/AtlasEngine.h

```diff
@@ -0,0 +1,759 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
It would definitely be helpful to break it up. I intend to do so in the future when I'm more certain about what's needed and what isn't. The other AtlasEngine is just as messy as the header file and needs a similar treatment.
It would be difficult for me to clean it up at this point already, because it's highly likely that I'll rewrite at least half of the code in the header file (anything related to the glyph cache basically).